presto-product-tests/src/main/java/io/prestosql/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -222,6 +222,16 @@ public void testAccessControlShowColumns()
         assertThat(bobExecutor.executeQuery(format("SHOW COLUMNS FROM %s", tableName))).hasRowsCount(2);
     }
 
+    @Test(groups = {AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
```

Comment:
Can you please write test in `io.prestosql.tests.AbstractTestQueries#testAccessControl` for that as well? Product test sounds like an overkill, but once you wrote it could stay here as well.

Reply:
Sure, will do :)